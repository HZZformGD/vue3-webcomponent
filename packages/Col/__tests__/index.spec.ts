import example from "../docs/demo.vue";
import { mount } from "@vue/test-utils";
test("test col example", () => {
  const wrapper = mount(example);
  expect(wrapper.html()).toMatchSnapshot();
  wrapper.unmount();
});

