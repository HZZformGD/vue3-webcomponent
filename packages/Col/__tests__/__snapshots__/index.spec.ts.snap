// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test col example 1`] = `
"<p>Row</p>
<div class=\\"flex\\">
  <xizi-col align=\\"center\\">
    <p>这是第一个</p>
    <p>这是第二个</p>
  </xizi-col>
</div>"
`;
